src/components/TemplateLibrary/index.js

```diff
@@ -138,6 +162,11 @@ const LibraryComponent = () => {
     return ciceroMark;
   };
 
+  /**
+   * Loads the template text, setup it and save the template to xml
```

Comment:
`Fetches templateIndex from https://templates.accordproject.org/, load the template, and save template details to CustomXML. `